weed/storage/store.go

```diff
@@ -557,6 +559,25 @@ func (s *Store) ConfigureVolume(i needle.VolumeId, replication string) error {
 		if err != nil {
 			return fmt.Errorf("volume %d failed to save vif: %v", i, err)
 		}
+                // update superblock
```

Comment:
please refactor this to a function, and just print a warning instead of failing in the middle.

Reply:
Ok, refactoring this into a function makes sense.
But I don't see why printing a warning here makes sense. If a problem occurs while loading or saving the ".vif", it fails with an error, so updating the superblock should be done the same way?

Reply:
there should be only one source of truth, which is the `.vif` file.